src/domain/descriptions/axisymmetric.cpp

```diff
@@ -120,7 +121,9 @@ std::shared_ptr<ablate::domain::Region> ablate::domain::descriptions::Axisymmetr
 
     // compute the outer shell start
     auto outerShellStart = numberCenterVertices + numberVerticesPerShell * (numberShells - 1);
-
+    // Coordinate used to track the z location
+    auto *coordinate = new PetscReal[]{0.0, 0.0, 0.0};
```

Comment:
This is being deleted anywhere?  Can we create it only on the stack?

Reply:
Good catch, I didn't realize I initialized it that way... It wasn't being deleted anywhere, but I fixed it to be created on the stack.